### ISSUE TITLE
What does "We fix the text encoder during training" mean? Does it mean that the parameters are not updated during training?

### ISSUE THREAD

**Issue by sunwhw** [raoyongming/DenseCLIP#33]
Thanks for your great work!  Does "We fix the text encoder during training" mean that the parameters of text encoder are not updated during training? If so, I can't find any "requires_grad=False" settings in the source code.

**Comment by sunwhw** [comment]
ok fine!
I think I find the answer.
Is this part?
"
optimizer = dict(type='AdamW', lr=0.0001, weight_decay=0.0001, 
        paramwise_cfg=dict(custom_keys={'backbone': dict(lr_mult=0.1),
                                        'text_encoder': dict(lr_mult=0.0),
                                        'norm': dict(decay_mult=0.)}))
"
 and Does "text_encoder': dict(lr_mult=0.0)" finish the work?

**Comment by raoyongming** [comment]
> ok fine! I think I find the answer. Is this part? " optimizer = dict(type='AdamW', lr=0.0001, weight_decay=0.0001, paramwise_cfg=dict(custom_keys={'backbone': dict(lr_mult=0.1), 'text_encoder': dict(lr_mult=0.0), 'norm': dict(decay_mult=0.)})) " and Does "text_encoder': dict(lr_mult=0.0)" finish the work?

Yes. We fix the text encoder by setting its learning rate to 0 which can be simply implemented with a single line of code as you mentioned.

**Comment by sunwhw** [comment]
Appreciate it.
